test/platform-checks/mzcompose.py

```diff
@@ -82,19 +81,21 @@ def create_mzs(
                 f"--var=default-storage-size={Materialized.Size.DEFAULT_SIZE}-1",
             ],
             volumes_extra=["secrets:/share/secrets"],
-            metadata_store="cockroach",
+            metadata_store="postgres-metadata" if postgres_consensus else "cockroach",
         )
     ]
 
 
 SERVICES = [
     TestCerts(),
-    # TODO(def-): Switch to CockroachOrPostgres after we have 4 versions
-    # support Postgres as metadata store
     Cockroach(
```

Comment:
This should use CockroachOrPostgres to allow switching more easily, tell me if I should make the switch.

Reply:
That would be great! Mind making the switch in a follow up PR? I want to get this one merged ASAP

Reply:
Works for me, will take a look tomorrow.